app/views/topics/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>New Topic</h1>
+
+<div class="row">
+  <div class="col-md-4">
+
+  </div>
+  <div class="col-md-8">
+    <%= form_for @topic do |f| %>
```

Comment:
restructured topics to be nested under user for new/create, edit/update and delete/destroy.  When creating a topic, the user is automatically supplied now.